README.md

```diff
@@ -72,8 +72,9 @@ Installation instructions for previous releases can be found in [Metrics Server
 
 Metrics Server | Metrics API group/version | Supported Kubernetes version
 ---------------|---------------------------|-----------------------------
-0.7.x          | `metrics.k8s.io/v1beta1`  | 1.19+
-0.6.x          | `metrics.k8s.io/v1beta1`  | 1.19+
+0.8.x          | `metrics.k8s.io/v1beta1`  | 1.31+
```

Comment:
I have updated last three referring to k8s versions we test against

https://github.com/kubernetes-sigs/metrics-server/blob/release-0.7/Makefile
https://github.com/kubernetes-sigs/metrics-server/blob/release-0.6/Makefile
https://github.com/kubernetes-sigs/metrics-server/blob/release-0.5/Makefile